products/BellHybrid/apps/application-bell-onboarding/presenter/OnBoardingShortcutsWindowPresenter.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "OnBoardingShortcutsWindowPresenter.hpp"
+#include "layouts/OnBoardingShortcutsLayoutProvider.hpp"
+
+namespace app::OnBoarding
+{
+    OnBoardingShortcutsWindowPresenter::OnBoardingShortcutsWindowPresenter(app::ApplicationCommon *app) : app(app)
+    {
+        initLayoutOptions();
+    }
+
+    std::vector<gui::Item *> OnBoardingShortcutsWindowPresenter::getLayouts() const
+    {
+        std::vector<gui::Item *> layouts;
+
+        for (auto const &option : layoutOptions) {
+            layouts.push_back(option);
+        }
+
+        return layouts;
+    }
+
+    gui::Item *OnBoardingShortcutsWindowPresenter::getFirstLayout() const
+    {
+        return layoutOptions.at(0);
+    }
+
+    bool OnBoardingShortcutsWindowPresenter::isLastLayout(const gui::Item *layout) const
+    {
+        return layoutOptions.back() == layout;
+    }
```

Comment:
`front()` would be enough to get the first element. If your intention is to check exceptions then `back()` should probably be replaced with `at()` to be consistent.

Reply:
Done